scenarios/aws/kindvm/run.go

```diff
@@ -154,3 +170,36 @@ agents:
 
 	return nil
 }
+
+func ConfigureECRCredentials(e resAws.Environment, vm *remote.Host, arch os.Architecture, opts ...pulumi.ResourceOption) (*goremote.Command, error) {
+	architecture := "x86_64"
+	if arch == os.ARM64Arch {
+		architecture = "aarch64"
+	}
+
+	unzipInstallCommand, err := vm.OS.PackageManager().Ensure("unzip", nil, "", opts...)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Unrelated, noticing now that calls to `Ensure` do not pin versions, might be something we work on

Reply:
I think replacing `unzip` by `unzip==<version>` could work. But would need to be tested

Reply:
I am not sure there is an added value in pinning versions, for two reasons:
- Versions are usually pinned by distro release (normally only minor/security changes).
- We don't pin AMIs and in most cases we don't even pin OS version.

Reply:
We do pin some AMIs, for example for the installer tests. We don't probably want to maintain all images and versions, but for folks who would like to, we might want to offer the option. No one ever asked for it, just noticing it now.